### PR TITLE
Adds BussIT as a service

### PR DESCRIPTION
**Pull request by Hoidi**

### CODE REVIEW

**Review by Portals — CHANGES_REQUESTED** [review]
Remove rdc upgrade. This should be done when 1.0 is ready instead

File: frontend/package.json [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
-    "version": "0.1.0",
+    "version": "1.0.0",
     "private": true,
     "dependencies": {
+        "@cthit/react-digit-components": "^1.0.1",
         "font-awesome": "^4.7.0",
-        "@cthit/react-digit-components": "^1.0.0-beta.55",
         "react": "^16.12.0",
-        "react-redux": "^7.1.3",
+        "react-redux": "^7.2.0",
```
Comment (Portals) [inline frontend/package.json]: Remove react-redux.

Comment (Portals) [reply frontend/package.json]: you only need to copy these:
https://github.com/cthit/react-digit-components/blob/master/package.json#L23

File: frontend/src/components/elements/service/index.jsx [hunk 2]
```diff
@@ -1,57 +1,58 @@
...
+    if (github_url !== "") {
+        return <a
+            style={{textDecoration: "none"}}
+            href={github_url}
+            target={"_blank"}
+        >
+            <DigitButton primary text={"GitHub"}/>
+        </a>
+    } else {
+        return <></>
```
Comment (Portals) [inline frontend/src/components/elements/service/index.jsx]:
```suggestion
        return null;
```

**Review by Portals — CHANGES_REQUESTED** [review]
Looks like you haven't run prettier either

File: frontend/src/components/elements/service/index.jsx (hunk 2 above)
Comment (lindskogen) [reply frontend/src/components/elements/service/index.jsx]: Make this a component instead?

File: frontend/package.json [hunk 3]
```diff
@@ -1,18 +1,15 @@
 {
     "name": "findit",
-    "version": "0.1.0",
+    "version": "1.0.0",
     "private": true,
     "dependencies": {
+        "@cthit/react-digit-components": "^1.0.1",
```
Comment (Portals) [inline frontend/package.json]: Upgrade to 1.0.2. Also check peer dependencies

File: frontend/src/components/elements/service/index.jsx [hunk 4]
```diff
@@ -1,57 +1,61 @@
 import React from "react";
 import "font-awesome/css/font-awesome.min.css";
-import { ServiceIcon } from "./styles";
+import {ServiceIcon} from "./styles";
```
Comment (Portals) [inline frontend/src/components/elements/service/index.jsx]: Looks like you have not run prettier

**Review by Portals — CHANGES_REQUESTED** [review]

File: frontend/src/components/elements/service/index.jsx (hunk 4 above)
Comment (Hoidi) [reply frontend/src/components/elements/service/index.jsx]: But I did run prettier and that's why there's change here 🤔

Comment (Portals) [reply frontend/src/components/elements/service/index.jsx]: Have you used the prettier config from cthit/style-guides?

Comment (Hoidi) [reply frontend/src/components/elements/service/index.jsx]: Yeah. Fixed it now. I formatted with ctrl+shift+alt+L but I should have used ctrl+shift+alt+P heheh

**Review by Portals — APPROVED** [review]

**Pull request merged**